homeassistant/components/binary_sensor/zwave.py

```diff
@@ -51,12 +51,19 @@ class ZWaveBinarySensor(BinarySensorDevice, zwave.ZWaveDeviceEntity):
     def __init__(self, value, device_class):
         """Initialize the sensor."""
         self._sensor_type = device_class
+        self._state = None
         zwave.ZWaveDeviceEntity.__init__(self, value, DOMAIN)
 
+    def update_properties(self):
+        """Callback on data changes for node values."""
+        self._state = self.get_value(
```

Comment:
Why not just `self._state = _value.data`
You are potentially bringing in data from another binary sensor in the same node.

Also everywhere else.